app/src/main/java/org/gdg/frisbee/android/api/PlusPersonDownloader.java

```diff
@@ -51,7 +51,12 @@ public Response intercept(Chain chain) throws IOException {
     }
 
     @Nullable
-    public Person getPersonSync(final String gplusId) {
+    public static Person getPersonSync(final String gplusId) {
```

Comment:
It could be names `getPersonSyncWithAppGplusClientInstance`. This makes sense if there are more instances.